testsuite/classlibrary/TestNode_Server.sc

```diff
@@ -0,0 +1,82 @@
+TestNode_Server : UnitTest {
+
+	var server;
+
+	setUp {
+		server = Server(this.class.name);
+		this.bootServer(server);
+	}
+
+	tearDown {
+		server.quit.remove;
+	}
+
+	test_get {
+		var setValue, getValue, node;
+
+		SynthDef(\test_get, { |control = 8| }).add;
+		server.sync;
+		setValue = 888;
+		node = Synth(\test_get, [control: setValue]);
+		server.sync;
+
+		getValue = 0;
+		node.get(\control, { |value|
+			getValue = value;
+		});
+		0.1.wait;
```

Comment:
Please change the `wait`s in this test and `test_getn` to `wait` on condition variables (see https://gist.github.com/brianlheim/91222d487afa18582c287b0a722ae272#avoid-use-of-wait)

Reply:
I'm not sure if the fix is as simple as what I just pushed (I haven't really understood the pitfalls of using Conditions is tests yet). Please advise.